srcpkgs/othman/template

```diff
@@ -1,9 +1,11 @@
 # Template file for 'othman'
 pkgname=othman
 version=0.5.1
-revision=1
+revision=2
+noarch=yes
 build_style=python2-module
-hostmakedepends="python-setuptools ImageMagick sqlite intltool"
+pycompile_module="othman"
+hostmakedepends="python ImageMagick sqlite intltool"
 depends="python-sqlite python-gobject amiri-font"
```

Comment:
`depends="gtk+3 python-gobject amiri-font"`

- `gtk+3` is missing (w/o it doesn't even launch)
- `python-sqlite` is not needed (we should actually remove it)